modular_skyrat/modules/cyborg/code/mechafabricator_designs.dm

```diff
@@ -38,3 +38,17 @@
 		/datum/material/plasma=7500,
 	)
 	category = list("Cyborg Upgrade Modules")
+
+/datum/design/inducer_upgrade
+	name = "Cyborg Upgrade (Inducer)"
+	id = "inducer_module"
+	construction_time = 60
+	build_type = MECHFAB
+	build_path = /obj/item/borg/upgrade/inducer
+	materials = list(
+		/datum/material/iron=10000,
+		/datum/material/gold=4000,
+		/datum/material/plasma=2000,
```

Comment:
What's the cost of a normal inducer, compared to this?

Reply:
The engineering inducers cannot be printed regularly and might not have a set material cost. I sort of guess with a material cost looking at other cyborg upgrades

Reply:
Science ones can, check that

Reply:
3000 iron, 1000 glass.